### PR TITLE
Mostly cosmetic docs cleanup

### PR DESCRIPTION
**Pull request by marians**
Done:
- added table of contents in README.md
- formatting
- set syntax highlighting explicitly
- corrected some language things

TODO:
- Clean up content under `4. Add new member to core members` so that it actually makes sense

Ping @JosephSalisbury @zeisss - I need some help with the open point above.

### CODE REVIEW

**Comment by marians** [comment]
Pinging @JosephSalisbury @zeisss again

File: docs/getting_started.md [hunk 1]
```diff
@@ -31,16 +31,16 @@ You now should find a `kocho` binary file in your current folder.
 
 ## Configuring Kocho
 
-First, let's initialize cloudconfig and cloudformation templates. By default
+First, let's initialize Cloud-Config and CloudFormation templates. By default
```
Comment (zeisss) [inline docs/getting_started.md]: not sure about this change. `Cloud-Config` looks weird.

**Comment by zeisss** [comment]
LGTM otherwise

File: docs/getting_started.md (hunk 1 above)
Comment (marians) [inline docs/getting_started.md]: I even looked up the official spelling

Comment (zeisss) [inline docs/getting_started.md]: Then I am just out of sync. Cool. Go ahead :)

**Comment by hectorj2f** [comment]
@marians Please rebase your PR to merge it

**Comment by marians** [comment]
Closing this, as I cannot work on it. Feel free to cherry-pick the changes.

**Pull request closed without merging**